editor/editor_audio_buses.cpp

```diff
@@ -461,10 +461,10 @@ void EditorAudioBus::drop_data(const Point2 &p_point, const Variant &p_data) {
 
 Variant EditorAudioBus::get_drag_data_fw(const Point2 &p_point, Control *p_from) {
 
-	print_line("drag fw");
+	print_line(TTR("drag fw"));
```

Comment:
Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.